.vscode/notebooks/endgame.github-issues

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "kind": 1,
+    "kind": 1,October 2023
```

Comment:
Is this intentional?

Reply:
Yeah our find replace support seems to have really messed something up here. Will see if I can create an issue

Reply:
Opened #194632 for replace issue. Manually doing replacements now